package Exception;

use Exception::Class ('Exception');

1;
